msi/GitProduct.wxs

```diff
@@ -284,6 +286,9 @@
       <Custom Action="ExecConfigureCoreAutoCrlf" After="ExecPostInstallBat">
         <![CDATA[REMOVE <> "ALL"]]>
       </Custom>
+      <Custom Action="ExecConfigureSslCAInfo" After="ExecConfigureCoreAutoCrlf">
+        <![CDATA[REMOVE <> "ALL"]]>
+      </Custom>
```

Comment:
I'm not sure what was "unexpected", but I also wanted to expand on my original comment  to help clarify. Perhaps I could have explained this a bit more in the commit message. The initial `<CustomAction />` element defines the custom action in the MSI `CustomAction` table. the `<Custom />` element later on "schedules" the defined custom action to run, either before or after another MSI installer action, or at a specific point within a sequence of actions to be performed by the installer.